<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>鐵路查詢表</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Refresh_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EXIT_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_edit_box">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GO_btn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="start_combo_box">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="arrival_combo_box">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="date_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>時間</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="start_text">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>起始</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="arrival_text">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>抵達</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="btn_group_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="THSR_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">台灣高鐵</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="KRTC_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>高雄捷運</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TRTC_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>台北捷運</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TRA_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>台灣鐵路</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>631</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
